### ISSUE TITLE
Account Fingerprint Phrase does not match in Windows portable desktop app

### ISSUE THREAD

**Issue by cksapp** [bitwarden/clients#5297]
### Steps To Reproduce

1. View Bitwarden account fingerprint phrase on any client such as the web-vault, as described in https://bitwarden.com/help/fingerprint-phrase/

2. Download and run the Windows portable desktop app of Bitwarden from https://vault.bitwarden.com/download/?app=desktop&platform=windows&variant=portable

3. Login to the portable desktop app with the same Bitwarden user account
4. View the **Account Fingerprint Phrase** in the portable app, which should be different than the recorded account's fingerprint phrase from all other client apps

### Expected Result

Account Fingerprint Phrases should match as this is used to verify the public key exchange between a user and Organization during onboarding as detailed in [BWN-01-008](https://cdn.bitwarden.net/misc/Bitwarden%20Security%20Assessment%20Report.pdf)

### Actual Result

Portable app's **Account's Fingerprint Phrase** does not match that of other clients, which could lead to confusion between Org admins and users

## Web-vault
![2023-04-27_09h49_04](https://user-images.githubusercontent.com/66083310/234888061-e30a0178-f96a-4bb9-a68b-6ae160d6306f.png)

## Standard Installer
![2023-04-27_09h49_23](https://user-images.githubusercontent.com/66083310/234888064-fdd450c0-d6a1-4c1d-a387-cf9a95ffec0f.png)

## Windows Portable
![2023-04-27_09h49_46](https://user-images.githubusercontent.com/66083310/234888067-0a646577-2a0c-4f16-b5fa-8555f44dab02.png)

### Additional Context

Original source: https://www.reddit.com/r/Bitwarden/comments/130kzbh/fingerprint_phrase_doesnt_match/

Tested and able to reproduce

### Operating System

Windows

### Operating System Version

10 Version 22H2 (OS Build 19045.2846)

### Installation method

Direct Download (from bitwarden.com)

### Build Version

2023.4.0

### Issue Tracking Info

- [x] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by SaabirahMohamed** [comment]
Hello @cksapp 

Thank you for your report! 

I was able to reproduce this issue, and I have flagged this to our engineering team.

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time - our engineering team will be happy to review these.

Thanks once again!

**Comment by tuanpham96** [comment]
I'm having the same issue but with Linux AppImage. 

```
Version 2023.5.1
Shell 24.1.1
Renderer 112.0.5615.50
Node 18.14.0
Architecture x64
```

Since I have my Linux desktop app auto-updated, I also tried to make sure that my current local AppImage's the sha256sum and sha512sum (`sha512sum Bitwarden.AppImage | cut -f1 -d\ | xxd -r -p | base64`) both match with the ones outlined in the [release page of V2023.5.1](https://github.com/bitwarden/clients/releases/tag/desktop-v2023.5.1), and they did. 

The account fingerprint on the web login is the same when I first created my account, let's say `A-B-C-D`, and it is also similar to my iOS device's app's fingerprint. Both `A-B-C-D`.

But not for my Linux desktop app, let's say `X-Y-Z-W`; I tried logging out of my Linux desktop app and it's still `X-Y-Z-W`.
